src/app/search/state/search.query.ts

```diff
@@ -68,11 +70,20 @@ export class SearchQuery extends Query<SearchState> {
       return true;
     } else if (workflows.length === 0 && param === 'workflow') {
       return false;
+    } else if (workflows.length > 0 && tools.length > 0) {
+      this.savedTabIndex$.subscribe(index => {
```

Comment:
Don't need to subscribe, you can do this.getValue().currentTabIndex

Reply:
Done, thanks for the tip!